fe1-web/src/features/rollCall/components/EventRollCall.tsx

```diff
@@ -57,53 +56,24 @@ const EventRollCall = (props: IPropTypes) => {
         );
         return;
       }
-      const time = Timestamp.EpochNow();
-      requestReopenRollCall(event.idAlias, time)
-        .then(() => {
-          if (!event.idAlias) {
-            console.debug(
-              'Unable to send roll call re-open request, the event does not have an idAlias',
-            );
-            return;
-          }
-          navigation.navigate(STRINGS.roll_call_open, {
-            rollCallID: event.id.toString(),
-            updateID: Hash.fromStringArray(
-              EventTags.ROLL_CALL,
-              lao.id.toString(),
-              event.idAlias.toString(),
-              time.toString(),
-            ),
-          });
-        })
-        .catch((e) => console.debug('Unable to send Roll call re-open request', e));
+      requestReopenRollCall(event.idAlias).catch((e) =>
+        console.debug('Unable to send Roll call re-open request', e),
```

Comment:
we also have toast messages (just search for "toast" in the codebase) to indicate to the user in a clearly visible way that something failed.
It's better than expecting the user to have the developer tools open 😄

Reply:
Right, toast seems a better idea !